deeposlandia/datasets/aerial.py

```diff
@@ -0,0 +1,144 @@
+"""Aerial dataset module
+
+Model Aerial images for convolutional neural network applications. Data is
+downloadable at https://project.inria.fr/aerialimagelabeling/files/.
+
+"""
+
+from multiprocessing import Pool
+import numpy as np
+import os
+from PIL import Image
+
+from deeposlandia.datasets import dataset
+from deeposlandia import utils
+
+
```

Comment:
The logger should be instantiated here. You don't have to use the one which is in the `utils` module.

Reply:
I do not understand this point. Do you suggest to duplicate the logger object as many times as the number of module?

I would have guessed that it is better to have one logger for the complete app.

Reply:
Ok thanks for the reference, I will read it and propose code updates!

Reply:
Done in a new commit (would have been better in another PR, anyway it is done!).